var/spack/repos/builtin/packages/py-tomli/package.py

```diff
@@ -12,24 +12,30 @@ class PyTomli(PythonPackage):
     Tomli is fully compatible with TOML v1.0.0."""
 
     homepage = "https://github.com/hukkin/tomli"
-    url = "https://files.pythonhosted.org/packages/py3/t/tomli/tomli-1.2.2-py3-none-any.whl"
-    list_url = "https://pypi.org/simple/tomli/"
+    pypi = "tomli/tomli-2.0.1.tar.gz"
+    git = "https://github.com/hukkin/tomli.git"
+
+    maintainers = ["charmoniumq"]
 
     version(
         "2.0.1",
-        sha256="939de3e7a6161af0c887ef91b7d41a53e7c5a1ca976325f429cb46ea9bc30ecc",
+        sha256="de526c12914f0c550d15924c62d72abc48d6fe7364aa87328337a31007fe8a4f",
         expand=False,
```

Comment:
You can drop `expand=False` if you're using `.tar.gz` files now. I think the reason we used to use wheels was because `py-flit-core` had a circular dep on tomli, but that's no longer the case, so I'm fine with this change.